src/main/java/org/springframework/integration/aws/config/xml/AWSNamespaceHandler.java

```diff
@@ -27,11 +26,12 @@
  * @author Artem Bilan
  * @since 0.5
  */
-public class AWSNamespaceHandler extends AbstractIntegrationNamespaceHandler {
+public class AwsNamespaceHandler extends AbstractIntegrationNamespaceHandler {
```

Comment:
@garyrussell , I see that the file hasn't been renamed.
Not sure what's wrong, but for me on Windows it looks good.
Any advise for renaming with case changes over a Git?

Thanks in advance!